storage/src/main/java/com/tencent/rss/storage/handler/impl/LocalFileClientReadHandler.java

```diff
@@ -21,127 +21,76 @@
 import java.util.List;
 
 import com.google.common.collect.Lists;
+import org.roaringbitmap.longlong.Roaring64NavigableMap;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.tencent.rss.client.api.ShuffleServerClient;
-import com.tencent.rss.client.request.RssGetShuffleDataRequest;
-import com.tencent.rss.client.request.RssGetShuffleIndexRequest;
-import com.tencent.rss.client.response.RssGetShuffleDataResponse;
 import com.tencent.rss.common.ShuffleDataResult;
 import com.tencent.rss.common.ShuffleDataSegment;
-import com.tencent.rss.common.ShuffleIndexResult;
 import com.tencent.rss.common.exception.RssException;
-import com.tencent.rss.common.util.RssUtils;
 
 public class LocalFileClientReadHandler extends AbstractClientReadHandler {
```

Comment:
there should be a new handler, eg, ReplicaReadHandler,  it should have the logic how to read data with replica > 1

Reply:
QuorumReadHandler is better ?

This patch already has a ReplicaReadHandler for each replica.